pages.pl/common/clojure.md

```diff
@@ -1,6 +1,6 @@
 # clojure
 
-> Te polecenie jest aliasem `clj`.
+> To poleceniejest aliasem `clj`.
```

Comment:
Missing space. Should be "polecenie jest".

Reply:
Fixed, sorry :)